README.md

```diff
@@ -13,35 +13,86 @@ method. For example, using [kind](https://kind.sigs.k8s.io/):
 kind create cluster --name xnat
 ```
 
-### Package the chart
+### Install the CNPG Operator
+
+We use the [CNPG Operator](https://github.com/cloudnative-pg/cloudnative-pg) to
+deploy Postgres. The operator can be installed using Helm:
+
+```bash
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+  --namespace cnpg-system \
+  --create-namespace \
+  cnpg/cloudnative-pg
+```
+
+### Create a namespace to install the chart
+
+Create a manifest for the namespace:
+
+```bash
+cat <<EOF > namespace.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: xnat-core
+EOF
+```
+
+Create the namespace:
+
+```bash
+kubectl apply -f namespace.yaml
+```
+
+### Create a secret containing Postgres credentials
+
+Create a manifest for the secret:
+
+```bash
+cat <<EOF > secret.yaml
+apiVersion: v1
+stringData:
+  username: xnat
+  password: xnat
+kind: Secret
+metadata:
+  name: pg-user-secret
```

Comment:
```suggestion
  name: pg-user-secret
  namespace: xnat-core
```

Reply:
Otherwise the secret would get installed in the `default` namespace.

Reply:
oh yep that makes it simpler (I was doing `kubectl apply -f secret.yaml -n xnat-core`)